all-in-one/jaeger-all-in-one-template.yml

```diff
@@ -52,7 +52,7 @@ items:
                   protocol: UDP
                 - containerPort: 5778
                   protocol: TCP
-                - containerPort: 16686
+                - containerPort: 14269
```

Comment:
You shouldn't replace the existing port here - instead add another entry for the admin port.